### PR TITLE
Rearranged the outline, added pdf using pandoc

### PR DESCRIPTION
**Pull request by sumitsahrawat**
I have re-ordered to content so that it all looks more logical.
The main motivation behind the reordering was that the table of content looked completely random when I first created the pdf.
Markdown looks filled with Bold fonts now, but the pdf looks great.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,50 +57,55 @@ $ sudo apt-get update
 $ sudo apt-get install cabal-install-1.20 ghc-7.8.3 happy-1.19.4 alex-3.1.3
 ```
 
-Then add
-`~/.cabal/bin:/opt/cabal/1.20/bin:/opt/ghc/7.8.3/bin:/opt/happy/1.19.4/bin:/opt/alex/3.1.3/bin` to your PATH (bash_profile, zshrc, bashrc, etc)
```
Comment (sumitsahrawat) [inline README.md]: Too long for the pdf, split into bullets.

File: README.md [hunk 2]
```diff
@@ -111,9 +127,11 @@ sudo yum-config-manager --add-repo https://copr.fedoraproject.org/coprs/petersen
 sudo yum install ghc cabal-install
```
Comment (sumitsahrawat) [inline README.md]: The line above this one is very long, still causes problems in the pdf.

Comment (fenak) [inline README.md]: could be splitted to:
```
sudo yum-config-manager --add-repo \
https://copr.fedoraproject.org/coprs/petersen/ghc-7.8.4/repo/fedora-21/petersen-ghc-7.8.4-fedora-21.repo 
```

Comment (sumitsahrawat) [inline README.md]: It still eats up the last ~15 chars. I'll have to reduce the margins to make it fit.

Comment (sumitsahrawat) [inline README.md]: Does not work even with the smallest of margins :frowning:

margins

Comment (fenak) [inline README.md]: Maybe:
```
sudo yum-config-manager --add-repo \
https://copr.fedoraproject.org/coprs/petersen/ghc-7.8.4/\
repo/fedora-21/petersen-ghc-7.8.4-fedora-21.repo
```
Will look horrible on web though...

**Comment by sumitsahrawat** [comment]
Check it out now. It looks good.

**Comment by fenak** [comment]
Looks awesome! :smile: :+1:

File: README.md [hunk 3]
```diff
@@ -46,50 +57,55 @@ $ sudo apt-get update
...
 
-Then add
-`~/.cabal/bin:/opt/cabal/1.20/bin:/opt/ghc/7.8.3/bin:/opt/happy/1.19.4/bin:/opt/alex/3.1.3/bin` to your PATH (bash_profile, zshrc, bashrc, etc)
+Then add the following to your `$PATH` (bash\_profile, zshrc, bashrc, etc):
+
+- `~/.cabal/bin`
+- `/opt/cabal/1.20/bin`
+- `/opt/ghc/7.8.3/bin`
+- `/opt/happy/1.19.4/bin`
+- `/opt/alex/3.1.3/bin`
```
Comment (bitemyapp) [inline README.md]: Why did you dismantle the path segment that could've been copied & pasted successfully? This won't work for anybody that doesn't know `PATH` in Unix shells.

Comment (sumitsahrawat) [inline README.md]: It was too long to fit in the width of a normal page. I'll try and see it it does now, as I've changed the pages to landscape mode.

**Comment by sumitsahrawat** [comment]
The `PATH` can be put on one line now. Changed. :+1:

**Comment by bitemyapp** [comment]
Wow.

**Comment by bitemyapp** [comment]
This is incredibly generous of both of you. I noticed the link for the dialogues doesn't work in the PDF, I'm going to fix that and make the other relative links absolute, if there are any.

**Comment by sumitsahrawat** [comment]
Glad I could help. :+1:

Btw, the dialogues link works for me (using evince to view the pdf). But it might not be the universal behavior.

You can link dialogues.md to the version online, and add a line stating that the file is also shipped with the repository.

**Comment by sumitsahrawat** [comment]
The link works only when the .md file is in the same directory as the .pdf file.

**Comment by bitemyapp** [comment]
@sumitsahrawat yeah, I'm just going to link it back to the Github Repo and mention where it goes.

**Pull request merged**